templates/minimal/README.md

```diff
@@ -45,65 +69,155 @@ git clone https://github.com/paritytech/polkadot-sdk-minimal-template.git minima
 cd minimal-template
 ```
 
-### Build
+## Starting a Minimal Template Chain
+
+### Omni Node
+
+[Omni Node](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/omni_node/index.html) can
+be used to run the minimal template's runtime. `polkadot-omni-node` binary crate usage is described at a high-level
+[on crates.io](https://crates.io/crates/polkadot-omni-node).
+
+#### Install `polkadot-omni-node`
+
+```sh
+cargo install polkadot-omni-node
+```
```

Comment:
You can simply say: Please see `crates.io/omni-node` for installation etc to reduce duplication, but this is also good. 

This is the great thing about having good READMEs, you don't have to repeat stuff :)) 

See more about this idea here: https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/meta_contributing/index.html#principles